app/src/main/java/de/grobox/transportr/utils/TransportrUtils.kt

```diff
@@ -114,3 +119,16 @@ object TransportrUtils {
 }
 
 fun Location.hasLocation() = hasCoord() && (latAs1E6 != 0 || lonAs1E6 != 0)
+
+fun updateGlobalHttpProxy(newProxy: Proxy, manager: TransportNetworkManager) {
```

Comment:
Might be better to move this inside the `TransportrUtils` object (static class).